StatBot/StatBot/CommandHandler.cs

```diff
@@ -3,41 +3,54 @@
 using System.Reflection;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.Extensions.DependencyInjection;
 using Discord;
 using Discord.Commands;
 using Discord.WebSocket;
 
 namespace StatBot
 {
+    /// <summary>
+    /// Handles command setup methods.
+    /// </summary>
     public class CommandHandler
     {
         private readonly DiscordSocketClient _client;
         private readonly CommandService _commands;
+        private readonly IServiceProvider _services;
         private Config _config;
 
-        // Retrieve client and CommandService instance via ctor
-        public CommandHandler(DiscordSocketClient client, CommandService commands, Config config)
+        /// <summary>
+        /// Command handler constructor.
+        /// </summary>
+        /// <param name="client">The discord client to be passed in for dependency injection.</param>
+        /// <param name="commands">The command service to be passed in for dependency injection.</param>
+        /// <param name="services">The service provider collection to be passed in for dependency injection.</param>
+        /// <param name="config">The config to be passed in for dependency injection.</param>
```

Comment:
nit: "to be passed in" not necessary when you're describing what parameters are. Maybe something like "Discord client connected to X", "Commands service instance for X", "Service provider collection for X". Try to think past the fact that you're using them for DI and ask what they're being used for once injected.

Reply:
actually in this vein @lsorell, I noticed in one of your descriptions in the sample docs markdown you shared, you wrote

"This command stores this id here" or something like that. More succinct and to the point would just be: "Stores this id here". Less can be more.